src/release.js

```diff
@@ -1,41 +1,39 @@
 'use strict'
 const github = require('@actions/github')
+const { isSomeCommitStale } = require('./time-utils.js')
+const { logWarning } = require('./log')
 
 async function getLatestRelease(token) {
-  const octokit = github.getOctokit(token)
-  const { owner, repo } = github.context.repo
+  try {
+    const octokit = github.getOctokit(token)
+    const { owner, repo } = github.context.repo
 
-  const { data } = await octokit.rest.repos.getLatestRelease({
-    owner,
-    repo,
-  })
+    const { data } = await octokit.rest.repos.getLatestRelease({
+      owner,
+      repo,
+    })
 
-  return data
+    return data
+  } catch (error) {
+    logWarning('No latest release found')
```

Comment:
let's do nothing here and keep the logging, for consistency, in the function which invokes this function

Reply:
I've added the logging here because eslint has empty block rule. I can ignore it if necessary

Reply:
just put a comment